File: backend-v2/src/main/java/vn/io/vutiendat3601/shop/v2/order/OrderItemDto.java
package vn.io.vutiendat3601.shop.v2.order;

import java.math.BigDecimal;

public record OrderItemDto(
    String orderTrackingNumber,
    int quantity,
    BigDecimal totalAmount,
    BigDecimal couponAmount,
    BigDecimal finalAmount,
    String productNo,
    String productName,
    String couponCode) {}

File: frontend-v2/src/app/domain/order/order-item-dto.ts
export class OrderItemDto {
  constructor(
    public orderTrackingNumber: string,
    public quantity: number,
    public totalAmount: number,
    public couponAmount: number,
    public finalAmount: number,
    public productNo: string,
    public productName: string,
    public couponCode: string
  ) {}
}
